website/docs/docs/dbt-cloud/on-premises/architecture.md

```diff
@@ -0,0 +1,108 @@
+---
+id: architecture
+title: Architecture & Data Flows
+---
+
+This guide is intended to help administrators running instances of dbt Cloud on-premises understand the internal components of their instance, as well as how their instance will interact with other services over the internet.
+
+## Required External Dependencies
+
+dbt Cloud has several external network dependencies that are required for normal operation. This section enumerates the required external dependencies for a normal dbt Cloud installation.
+
+### Replicated
+
+Replicated (https://www.replicated.com/) is a third-party service that provides a management layer over the dbt Cloud Kubernetes appliance. Our installer configures an additional KOTS (https://kots.io/) appliance to configure and manage the dbt Cloud appliance. KOTS is open-source, Apache 2 licensed software developed by Replicated.
+
+An overview of Replicated's security posture can be found at: https://www.replicated.com/resources/files/Replicated-Security-Whitepaper.pdf
+
+#### Initial Installation
+
+During initial installation, the KOTS appliance can be directly downloaded from the Internet or packaged up and delivered to your infrastructure. Access to the following resources is required:
+
+##### Accessed via HTTPS
+
+- `get.replicated.com`: This endpoint hosts the Replicated install script.
+- `registry.replicated.com`: The private Replicated Docker registry.
+- `registry-data.replicated.com`: The private Replicated Docker registry.
+- `quay.io`: Some dependencies of Replicated are hosted as public images in the Quay.io registry.
+- `hub.docker.com`: Some dependencies of Replicated are hosted as public images in Docker Hub.
+
+Replicated maintains a list of Replicated-owned IPs for whitelisting purposes at https://github.com/replicatedhq/ips/blob/master/ip_addresses.json.
+
+#### Application Installation and Upgrades
+
+To install your application and perform updates, some external connections are required. All connections are initiated from inside the network, and can vary depending on the installation method and the application update.
```

Comment:
What is the different between the initial installation and installing the application?

Reply:
@amychen1776 I've updated the language to make it more clear what the differences are